fiesta/apps/notifications/services/membership.py

```
@@ -0,0 +1,94 @@
+def notify_new_membership(membership: SectionMembership) -> None:
+    """
+    Called (via on_commit) when a new SectionMembership is created in UNCONFIRMED state.
+
+    1. Send confirmation email to the new applicant (if section config allows).
+    2. Send/enqueue digest email to all editors/admins (if section config allows).
+    """
+    section = membership.section
+
+    try:
+        config = section.sections_plugin_configuration
+    except Exception:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)